services/api/src/api/routes/webhook.py

```diff
@@ -25,12 +25,16 @@
             "properties": {
                 "headSha": {"type": "string"},
                 "name": {"type": "string"},
-                "type": {"type": "string", "enum": ["dataset", "model", "space"]},
+                "type": {
+                    "type": "string",
+                    "enum": ["repo", "repo.content", "repo.config", "discussion", "discussion.comment"],
```

Comment:
it will break if another scope is added, no?

Reply:
maybe just let it as "string"

Reply:
alternative: only list the accepted values here: if another one, such as "discussion" appears, the JSON validation will fail, and the webhook will be discarded

Reply:
also: is "scope" always present (even in "delete")?

Reply:
this one was a bad copy paste